### PR TITLE
[docs] PR for Document PLAT-1960 Part 2 Alerting Enhancements (Doc-PLAT-2399)

### PR DESCRIPTION
**Pull request by lizayugabyte**
Documentation to address https://yugabyte.atlassian.net/browse/PLAT-2399

Prometheus is described in latest/yugabyte-platform/troubleshoot/universe-issues/#command-line-access-to-metrics
Federated is addressed via a link to the blog.

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md [hunk 1]
```diff
@@ -12,26 +12,32 @@ isTocNested: true
 showAsideToc: true
 ---
 
-Yugabyte Platform can check universes for issues that may affect deployment. Should problems arise, Yugabyte Platform can automatically issue alert notifications.
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md:15]: Accidental removal?

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md [hunk 2]
```diff
@@ -41,35 +47,94 @@ Regardless of the alert level, you create and configure an alert as follows:
...
+
+  <br><br>
+
+- To create a new channel, click **Add Channel** and then complete the **Create new alert channel** dialog shown in the following illustration:<br>
+
+  ![New channel](/images/yp/config-alerts6.png)
+
+  <br><br>
+
+  If you select **Email** as a notification delivery method, perform the following:
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md:78]: "do the following" is shorter, and more non-native speaker friendly...

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md:15]: This is weird.  
Preview:
Screen Shot 2021-12-02 at 8 31 00 AM

Localhost:
Screen Shot 2021-12-02 at 8 31 13 AM

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md:78]: As a non-native speaker, I disagree with the second argument: to some of us, "do" sounds like a command

**Review by anmalysh-yb — APPROVED** [review]
LGTM

**Pull request merged**